vm/rust/src/lib.rs

```diff
@@ -425,6 +425,8 @@ lazy_static! {
     static ref CONSTANTS: HashMap<String, VersionedConstants> = {
         let mut m = HashMap::new();
         m.insert("0.13.0".to_string(), serde_json::from_slice(include_bytes!("../versioned_constants_13_0.json")).unwrap());
+        m.insert("0.13.1".to_string(), serde_json::from_slice(include_bytes!("../versioned_constants_13_1.json")).unwrap());
```

Comment:
Should this not be `0.13.1.1`?

Reply:
I was checking this commit - https://github.com/starkware-libs/blockifier/commit/dfea574bb346ec43de1a4a54e9a44366cecd0a6d#diff-79227bb6cc8b8dd82bfcc791f0a911f73dcfec44d04b56e7dfc940c78311e76c this file (0.13.1) is actually missing right now. Here is how we use it:
```
CONSTANTS.get(&version_str.to_string()).unwrap_or(VersionedConstants::latest_constants()).to_owned()
```
in case version_str file is present we use it's values otherwise we use latest VerionedConstants from blockifier (which is 0.13.1.1 in this PR)

Reply:
I removed this file from the PR